packages/rocketchat-apps/client/orchestrator.js

```diff
@@ -0,0 +1,83 @@
+import { AppWebsocketReceiver } from './communication';
+
+class AppClientOrchestrator {
+	constructor() {
+		this.ws = new AppWebsocketReceiver(this);
+
+		this._addAdminMenuOption();
+
+		const loadLangs = setInterval(() => {
```

Comment:
Hm... is there another way other than setting an interval?  I thought meteor had a way to add a callback when its ready.